tests/acceptance/let-helper-test.js

```diff
@@ -94,6 +94,22 @@ describe('Acceptance: let helper', function() {
         });
       });
 
+      it('scopes to outmost scope', () => {
+        andThen(() => {
```

Comment:
Ah, I see. I've added this PR which rewrites the test suite in BDD style.

https://github.com/thefrontside/ember-let/pull/10